### PR TITLE
Ensure issue templates include labels

### PR DESCRIPTION
**Pull request by maxkadel**
Connected to pulibrary/dacs_handbook#170

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/accessibility.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Accessibility
 about: Report an accessibility issue
 title: ''
-labels: accessibility general
+labels: 'accessibility-general'
```
Comment (christinach) [inline .github/ISSUE_TEMPLATE/accessibility.md:5]: @maxkadel can you change this to 'accessibility-awaiting-triage' it's the label that we should use for new accessibility tickets if we have not triaged or identify yet the accessibility level issue

**Review by christinach — CHANGES_REQUESTED** [review]

**Review by sandbergja — APPROVED** [review]
Thanks, @maxkadel ! 🦄

File: .github/ISSUE_TEMPLATE/accessibility.md (hunk 1 above)
Comment (christinach) [reply .github/ISSUE_TEMPLATE/accessibility.md:5]: 'accessibility-general' is the label we use when we have triaged the ticket and we have not yet identified the level of the accessibility issue or there is no one to identify .

Comment (maxkadel) [reply .github/ISSUE_TEMPLATE/accessibility.md]: Thanks, I was wondering about that. Changed.

**Review by christinach — APPROVED** [review]
Thanks @maxkadel 🦄

**Pull request merged**